data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+ <title xml:lang="lat">Onomasticon</title>
+ <author>Julius Pollux</author>
+<editor>Frederick Hultsch</editor>
+ <funder>Harvard Library</funder>
+ <principal>Gregory Crane</principal>
+ <respStmt>
+  <persName xml:id="DDD">Digital Divide Data</persName>
+  <resp>Corrected and encoded the text</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Gregory Crane</persName>
+  <resp>Editor-in-Chief, Perseus Digital Library</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Matt Munson</persName>
+  <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Annette Gessner</persName>
+  <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Thibault Clérice</persName>
+  <resp>Lead Developer (University of Leipzig) 2015 - 2017</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Bruce Robertson</persName>
+  <resp>Technical Advisor (Mount Allison University)</resp>
+ </respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>Harvard College Library</authority>
+<idno type="filename">tlg0542.tlg001.1st1K-grc1.xml</idno>
+<availability>
+ <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International License</p>
+</availability>
+<date>2018/</date>
+<publisher>Harvard College Library</publisher>
+<pubPlace>United States</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+ <biblStruct>
+ <monogr>
+  <title xml:lang="lat">Metrologicorum Scriptorum</title>
+  <editor>
+	<persName>
+	 <name>Frederick Hultsch</name>
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg0542">Julius Pollux</author>
+  <imprint>
+	<publisher>Teubner</publisher>
+	<pubPlace>Leipzig</pubPlace>
+	<date>1864</date>
+  </imprint>
+  <biblScope unit="volume">I</biblScope>
+ </monogr>
+ </biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+ <p>Text encoded in accordance with the latest EpiDoc standards</p>
+ <refsDecl n="CTS">
```

Comment:
The CTS mentions `book` and `chapter`, the text has `book` and `section`. And I think one level should be sufficient.